xml/common_intro_support.xml

```diff
@@ -29,7 +29,7 @@
  <para>
   If you are entitled to support, find details on how to collect information
   for a support ticket at
-  <link xlink:href="https://documentation.suse.com/sles-15/html/SLES-all/cha-adm-support.html"/>.
```

Comment:
Hi Souvik, please don't change links - as I explained in DOCTEAM-1151, those are OK as they are.